File: cpp/program13.cpp
// program38.c

#include <iostream>
using namespace std;

void DisplayDigit(int iNo)
{
    int iCnt = 0;

    for(iCnt = iNo; iCnt >= 1; iCnt-- )
    {
        cout<<iCnt<<"\n";
    }
}

int main()
{
    int iValue = 0;

    cout<<"Enter number"<<"\n";
    cin>>iValue;

    DisplayDigit(iValue);

    return 0;
}

File: java/program13.java
import java.util.*;

class program13
{
    public static void DisplayDigits(int iNo)
    {
        for(int icnt = 1; icnt <= iNo; icnt++)    
        {
            System.out.print(icnt+"\t");
        }
    }

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Frequency");
        int iValue = sobj.nextInt();

        DisplayDigits(iValue);
    }
}